tests/lib/config.js

```diff
@@ -28,6 +28,9 @@ var userAgent = {
 // set default viewport size to desktop
 casper.options.viewportSize = viewport.desktop;
 
+// set default wait timeout
+casper.options.waitTimeout = 5000;
```

Comment:
Figured it couldn't hurt, I guess the default Casper uses could always change in future versions?